### DISCUSSION TITLE
Grafana dashboards for pgSCV.

### DISCUSSION THREAD

**Discussion by lesovsky** [weaponry/pgscv#3] [General]
There are several dashboards are available for Grafana:
- [postgresql](https://grafana.com/grafana/dashboards/14540)
- [pgbouncer](https://grafana.com/grafana/dashboards/14541)
- [system](https://grafana.com/grafana/dashboards/14539)

Here you can leave feedback or suggest ideas.

**Comment by Tri0L** [comment]
**pgSCV: PostgreSQL**
Longest activity (Stat widget)
Dashboard query `max(max_over_time(postgres_activity_max_seconds{instance="$instance",service_id="$postgres"}))` 
Error: `"1:19: parse error: expected type range vector in call to function \"max_over_time\", got instant vector"`
Working query: `max(max_over_time(postgres_activity_max_seconds{instance="$instance",service_id="$postgres"}[1m]))`

**Reply by lesovsky** [reply]
Thanks, such syntax (omitting range duration) is allowed in MetricsQL which I prefer personally, but is not allowed in PromQL. Thank you for reporting. I removed usage of `max_over_time()`, this is copy-paste from my another dashboard and users might not need this hardcoded aggregation over `1m`. New revision has been uploaded.

**Comment by lesovsky** [comment]
The dashboards have been re-upload. New revisions have reviewed queries and fixed reported bugs.

**Comment by cloud-66** [comment]
Thank you for exporter and dashboards. I would suggest to add overal , non idle cpu utilization line to graph. Now it is necessary to summarize utilizations lines or explore cpu usage overview.
